src/test/java/org/pih/petl/job/SqlServerImportJobTest.java

```diff
@@ -28,12 +29,53 @@
         SpringRunnerTest.setupEnvironment();
     }
 
+    @After
+    public void dropTablesInTargetDB() throws Exception{
+        ApplicationConfig appConfig = etlService.getApplicationConfig();
+        EtlDataSource sqlServerDataSource = appConfig.getEtlDataSource("sqlserver-testcontainer.yml");
+        try (Connection c = DatabaseUtil.openConnection(sqlServerDataSource)) {
+            DatabaseUtil.dropTable(c, "encounter_types");
+        }
+    }
+
     @Test
     public void testLoadingFromMySQL() throws Exception {
         etlService.executeJob("sqlserverimport/job.yml");
+        verifyTableExists("encounter_types");
+        verifyRowCount("encounter_types", 62);
+
+        // by default, table should be dropped and recreated on each run, so consecutive runs should return the same result
+        etlService.executeJob("sqlserverimport/job.yml");
+        verifyTableExists("encounter_types");
+        verifyRowCount("encounter_types", 62);
+    }
+
+
+    @Test
```

Comment:
Added this test for the dropAndRecreateTable functionality I added last week (also changed the above test to confirm that is *does* drop and recreate by default)